dpwo.py

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
-import subprocess , argparse , sys , importlib.util , os
+from interfaces.PluginInterface import PluginInterface
+from typing import Type
+import plugins.PluginLoader as PluginLoader
+import subprocess , argparse , sys
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Consider possible security implications associated with subprocess module. https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b404-import-subprocess